genie/process_mutation.py

```diff
@@ -202,12 +202,14 @@ def process_mutation_workflow(
         "syn22053204",
         ifcollision="overwrite.local",
         downloadLocation=genie_config["genie_annotation_pkg"],
+        version=1,  # TODO: This should pull from a config file in the future
     )
     # Genome Nexus Jar file
     syn.get(
         "syn22084320",
         ifcollision="overwrite.local",
         downloadLocation=genie_config["genie_annotation_pkg"],
+        version=13,  # TODO: This should pull from a config file in the future
```

Comment:
@rxu17 I'd comment this out for now and just rely on the latest version until we are actually able to pull this from a config file.